docusaurus/docs/iOS/swiftui/components/typing-indicators.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Typing Indicators
+---
+
+## Typing Indicators Overview
+
+The SwiftUI SDK has support for typing indicators, shown when other participants in a conversation are typing. There are two places where the typing indicators are shown. The first one is when the user is in the channels list - when someone in a channel is typing, the indicator is shown in the corresponding channel list item. The second place, where a typing indicator is shown, is in the chat channel view. Depending on the configuration provided, the typing indicator can be shown either in the navigation bar, or above the composer, as an overlay over the message list. 
```

Comment:
This is again only a suggestion and very subjective but here are the veeeery minor details I'd change. Feel free to ignore those and just keep it the way you have it.

```suggestion
The SwiftUI SDK has support for typing indicators which are shown when other participants in a conversation are typing. 

There are two places where the typing indicators are shown. The first one is when the user is in the channels list - when someone in a channel is typing, the indicator appears in the corresponding channel list item. 
The second place, where a typing indicator is shown, is in the chat channel view. Depending on the configuration provided, the typing indicator can be shown either in the navigation bar or above the composer, as an overlay over the message list. 
```